kolibri/content/test/test_content_app.py

```diff
@@ -580,6 +583,28 @@ def test_parent_query_cache_hit(self):
             self.client.get(self._reverse_channel_url("contentnode-list"), data={"parent": id})
             self.assertFalse(mock_cache_set.called)
 
+    def test_in_lesson_filter(self):
+        facility = Facility.objects.create(name="MyFac")
+        admin = FacilityUser.objects.create(username="admin", facility=facility)
+        admin.set_password(DUMMY_PASSWORD)
+        admin.save()
+        nodes = []
+        nodes.append(content.ContentNode.objects.get(title='c3c1'))
```

Comment:
We should change that fixture JSON. The tests are hard for me to read because of these title.

Reply:
would it make it easier if I `get` by the `id` instead of the `title`?

Reply:
Note: would need to rewrite all the historic schema fixtures too (shouldn't be too hard, but the data needs to be consistent).

Reply:
I don't think you need to change the `.get`, because there is  nothing special about `nodes`. I'm just saying that the fixture seems like a hodgepodge of things and could have fields that are easier to understand. For example, take this channelmetadata model:

```
  {
    "fields": {
      "name": "testing",
      "description": "test data",
      "thumbnail": "",
      "author": "eli",
      "version": 0,
      "min_schema_version": "1",
      "root": "da7ecc42e62553eebc8121242746e88a"
    },
    "pk": "6199dde695db4ee4ab392222d5af1e5c",
    "model": "content.channelmetadata"
  },

```

Reply:
We will add updating content fixtures and tests to tech debt.